client/src/__tests__/__components__/first.test.js

```diff
@@ -0,0 +1,46 @@
+import 'babel-polyfill';
+import React from 'react';
+// import expect from 'expect';
+// import { expect } from 'chai';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import sinon from 'sinon';
+import { shallow, mount } from 'enzyme';
+import Signup from '../../components/groups/Notification';
+import AuthHeader from '../../components/headers/AuthHeader';
+
+const mockStore = configureStore();
+/* global jest */
+/* global expect */
+
+Object.defineProperty(window, 'sessionStorage', { value: jest.fn() });
+window.sessionStorage.getItem = jest.fn();
+window.sessionStorage.setItem = jest.fn();
+
+describe('<First/>', () => {
+  // before((done)=>{
```

Comment:
You should remove this commented code if you are not using it again. You can check through your codes and remove them where applicable. It's going to make your code more readable and look more professional